src/pages/api/[profile]/graphql.js

```diff
@@ -0,0 +1,164 @@
+import { getServerSession } from "@dbcdk/login-nextjs/server";
```

Comment:
/** @file - this file is a serverside 'proxy' for fbi-api .... eller hvad det nu er - ihvertfald en beskrivelse af hvorfor  filen er der og hvad den skal gøre godt for